lib/internal/streams/destroy.js

```diff
@@ -108,7 +108,7 @@ function undestroy() {
   }
 }
 
-function errorOrDestroy(stream, err) {
+function errorOrDestroy(stream, err, sync) {
```

Comment:
Shouldn't this be `sync` -> `async` or the condition below reversed considering if this arg is true we will emit on the nextTick and **not** immediately which contradicts the `sync` name?

Reply:
This is the way it’s done in other places. I think we should be consistent.